docs/simplesamlphp-reference-idp-hosted.txt

```diff
@@ -224,6 +224,10 @@ The following SAML 2.0 options are available:
 
 :   Note that this requires a configured memcache server.
 
+`saml20.ecp`
+:   Set to `TRUE` to enable the IdP to recieve authnrequests and send responses according the Enhanced Client or Proxy (ECP) Profile.
+    Defaults to `FALSE`.
+
 `saml20.hok.assertion`
 :   Set to `TRUE` to enable the IdP to send responses according the [Holder-of-Key Web Browser SSO Profile](./simplesamlphp-hok-idp).
     Defaults to `FALSE`.
```

Comment:
I think we need some comments in the IdP documentation about ECP. Basically, we need to explicitly tell that any **authentication filters that require interaction with the user will not work with ECP**, and therefore they must be disabled. The *consent* module is a good example of this. If it is enabled, authentication will not work unless the user gave consent for the SP on beforehand.